File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.py
c = int(input())

for i in range(c):
    n = input().split()
    tot=0
    co=0
    for j in range(1,len(n)):
        tot+=int(n[j])
    avg=tot/(len(n)-1)
    for k in range(1,len(n)):
        if int(n[k])>avg :
            co+=1
    upavg = (co/(len(n)-1))*100
    print("%.3f"%(upavg), end="%\n")

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.java
import java.util.Scanner;

public class Main {
	public static void main(String args[]) {
		Scanner sc = new Scanner(System.in);
		int C = sc.nextInt();
		for(int i=0; i<C; i++) {
			int tot =0;
			int N =sc.nextInt();
			int score[] = new int[N];
			for(int j=0; j<N; j++) {
				int co=0;
				score[j] = sc.nextInt();
				tot+=score[j];
				if(j==N-1) {
					double avg = tot/N;
					for(int k=0; k<N; k++) {
						if(score[k]>avg) {
							co++;
						}
					}
					System.out.printf("%.3f",((float)co/N)*100);
					System.out.println("%");
				}
			}
		}
	}
}
